esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2527,6 +2527,7 @@ bool CFileSprayEx::onCopy(IEspContext &context, IEspCopy &req, IEspCopyResponse
         wuFSpecDest->setFileMask(fileMask.str());
         wuOptions->setOverwrite(req.getOverwrite());
         wuOptions->setPreserveCompression(req.getPreserveCompression());
+        wuOptions->setExpireDays(req.getExpireDays());
```

Comment:
@wangkx you should provide a default or only set here if the value was set.  There should be a generated method for checking.